docs/configuration.asciidoc

```diff
@@ -314,6 +314,8 @@ Possible options are: `off`, `all`, `errors`, and `transactions`.
 
 The recorded body will be truncated if larger than 2 KiB.
 
+If the body parsing middleware captures the body as raw `Buffer` data, the request body will be represtend as the string `"<Buffer>"`.
```

Comment:
typo

```suggestion
If the body parsing middleware captures the body as raw `Buffer` data, the request body will be represented as the string `"<Buffer>"`.
```

Reply:
I think you resolved this without taking the fix. ;)

Reply:
Fixed it locally, but didn't push 🤦